Fix issues and add tests

akshitsethi opened the pull request:
This PR is work in progress and changes the following:
- Fix known issues.
- Add namespaces to the theme to know about performance bottlenecks.
- Add more tests and improve code coverage.

dz0ny commented on framework/classes/customizer/rearrange.php:
```diff
@@ -0,0 +1,126 @@
+            	'cf_hc_section' => 5,
+        		'cf_lb_section' => 10,
+            	'cf_hm_section' => 15,
+            	'cf_fb_section' => 20
```
can you make a comment what there are please

dz0ny commented on framework/classes/homepage_control_customizer_control.php:
```diff
@@ -0,0 +1,172 @@
+
+    use WP_Customize_Control;
+
+	if ( ! class_exists( 'Homepage_Control_Customizer_Control' ) && class_exists( 'WP_Customize_Control' ) ) :
```
you can remove class_exists

akshitsethi commented:
@dz0ny I have implemented the suggestions and also fixed a couple of issues which popped up during testing. This PR seems to be stable on my local server. I still need to test it on the `docker` version though.
